architecture/core_objects/builds.adoc

```diff
@@ -172,40 +174,20 @@ can also add authentication entries to this file by running the `docker login`
 command. The file will be created if it does not exist.
 
 Kubernetes provides the
-https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/design/secrets.md[Secret]
-resource, which is used to store your configuration and passwords. You must
-first create the `*Secret*` before builds can use your *_.dockercfg_* file for
-pushing the output image:
+-https://github.com/GoogleCloudPlatform/kubernetes/blob/master/docs/design/secrets.md[Secret]
+-resource, which is used to store your configuration and passwords. 
 
-====
-----
-$ osc create -f secret.json
-----
-====
-
-The `*data*` field for the `*Secret*` object must contain the `*dockercfg*` key
-with the value set to the base64-encoded content of the *_.dockercfg_* file. For
-example:
+To create the `*Secret*` resource from your local `.dockercfg` file, you can run
+following command:
 
 ====
-
 ----
-{
-  "apiVersion": "v1beta3",
-  "kind": "Secret",
-  "metadata": {
-    "name": "dockerhub"
-  },
-  "data": {
-    "dockercfg": "6yJodHRwc1ovL2zuZGV4LmRvY21lci5aby92MS8iOnsiYXV0aCI6ImJXWnZhblJwYXpwdVoybGxkR2d4TUE9PSIsImVtYWlsIj8ibWlAbWlmby5zayJ9fQ=="
-  }
-}
-
+$ openshift ex bundle-secret dockerhub ~/.dockercfg | osc create -f -
```

Comment:
little more detail on what this does please...
"generates json describing a secret named 'dockerhub'  and then passes it to osc create which will define the new secret object in your current namespace"